monai/transforms/croppad/array.py

```diff
@@ -967,15 +967,21 @@ class ResizeWithPadOrCrop(Transform):
             ``"median"``, ``"minimum"``, ``"reflect"``, ``"symmetric"``, ``"wrap"``, ``"empty"``}
             One of the listed string values or a user supplied function for padding. Defaults to ``"constant"``.
             See also: https://numpy.org/doc/1.18/reference/generated/numpy.pad.html
+        method: {``"symmetric"``, ``"end"``}
+            Pad image symmetric on every side or only pad at the end sides. Defaults to ``"symmetric"``.
```

Comment:
Pad image symmetrically
Please also modify the same place in `SpatialPad`

Reply:
Hi @yiheng-wang-nv ,

Thanks for your review.
I think we already have this arg in `SpatialPad`.

Thanks.

Reply:
updated in the latest commit.

Thanks.